inputfiles/idl/WebRTC.widl

```diff
@@ -0,0 +1,39 @@
+enum RTCPriorityType {
```

Comment:
this file should be added to https://github.com/Microsoft/TSJS-lib-generator/blob/master/inputfiles/idlSources.json, so that next fetch would bring it too.

Reply:
ok